README.md

```diff
@@ -293,6 +294,21 @@ It can also create [JEP 238](http://openjdk.java.net/jeps/238) multi-release jar
 `AtomicXxxFieldUpdater` for JDK<=8 and `VarHandle` for for JDK9+ if you 
 set `variant` to `"BOTH"`.
 
+### Arrays of atomic values
+
+You can declare arrays of all supported atomic value types. 
+By default arrays are transformed into corresponding `java.util.concurrent.atomic.Atomic*Array`.
+
+If you configure `variant = "VH"` an array will be transformed to plain array using [VarHandle](http://download.java.net/java/jdk9/docs/api/java/lang/invoke/VarHandle.html) to support atomic operations.
+  
+```kotlin
+val a = AtomicArray<T>(size) // similar to Array constructor
```

Comment:
^^ Nope, that does not work. Use `atomicArrayOfNulls<T>(size)` for this example.